uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -221,7 +221,7 @@ private fun FeedScreen(
                         onFavoriteChange = onFavoriteChange,
                         listState = tabLazyListStates.getValue(selectedTab),
                         onClickPlayPodcastButton = onClickPlayPodcastButton,
-                        isRevealed = scaffoldState.isRevealed,
+                        showItemCount = scaffoldState.isRevealed || filters.filterFavorite,
```

Comment:
Thank you 👍 
How about simply using filters.filterFavorite? I thought it was easy to understand that it was filtering.

Reply:
When i saw figma, item count is displayed when not filtered.
So i left it that use "isRevealed" condition.

Of course as you say, just using filters.filterFavorite is more simple.
But does it meet the design and specification?
If it satisfy the requirement, I will simplify this!!!

Reply:
I will talk with the designer for a moment. 👀

Reply:
I talk to the designer! Please just use `filters.filterFavorite` 🙏

Reply:
got it.
I'll fix code!